flaml/automl.py

```diff
@@ -2418,6 +2424,7 @@ def custom_metric(
 
         self._state.fit_kwargs = fit_kwargs
         custom_hp = custom_hp or self._settings.get("custom_hp")
+        self._skip_transform = self._settings.get("skip_transform") if skip_transform is None else skip_transform
```

Comment:
```suggestion
        self._skip_transform = skip_transform or self._settings.get("skip_transform")
```

Reply:
@liususan091219 See my comment in https://github.com/microsoft/FLAML/pull/665#discussion_r938235656

Reply:
resolved.